doc/tutorials/enable_s5.rst

```diff
@@ -91,26 +91,32 @@ Enable S5
 
 The procedure for enabling S5 is specific to the particular OS:
 
-* For Linux (LaaG) or Windows (WaaG), refer to the following configurations in the
-  ``misc/config_tools/data/sample_launch_scripts/nuc/launch_uos.sh`` launch script for ``acrn-dm``.
+* For Linux (LaaG) or Windows (WaaG), include these lines in the launch script:
 
-  .. literalinclude:: ../../../../misc/config_tools/data/sample_launch_scripts/nuc/launch_uos.sh
-     :name: laag-waag-script
-     :caption: LaaG/WaaG launch script
-     :linenos:
-     :lines: 97-117
-     :emphasize-lines: 2-4,17
-     :language: bash
+  .. code-block:: bash
 
-* For RT-Linux, refer to the ``misc/config_tools/data/sample_launch_scripts/nuc/launch_hard_rt_vm.sh`` script:
+     #for pm by vuart setting
+     pm_channel="--pm_notify_channel uart "
+     pm_by_vuart="--pm_by_vuart pty,/run/acrn/life_mngr_"$vm_name
+     pm_vuart_node=" -s 1:0,lpc -l com2,/run/acrn/life_mngr_"$vm_name
 
-  .. literalinclude:: ../../../../misc/config_tools/data/sample_launch_scripts/nuc/launch_hard_rt_vm.sh
-     :name: rt-script
-     :caption: RT-Linux launch script
-     :linenos:
-     :lines: 42-58
-     :emphasize-lines: 2-3,13
-     :language: bash
+     acrn-dm -A -m $mem_size -s 0:0,hostbridge \
+       ...
+       $pm_channel $pm_by_vuart $pm_vuart_node \
+       ...
+
+* For RT-Linux, include these lines in the launch script:
+
+  .. code-block:: bash
+
+     # for pm setting
+     pm_channel="--pm_notify_channel uart "
```

Comment:
is that trailing space inside the double quote needed?

Reply:
I don't think so (not tested it though ;-)... I think it's more of an extra safety measure. I'll remove them and also put each of those arguments on a separate line, that should take care of the need to separate them.